lib/athens/configuration.rb

```diff
@@ -1,15 +1,17 @@
 module Athens
   class Configuration
-    attr_accessor :aws_access_key, 
+    attr_accessor :aws_access_key,
                   :aws_secret_key,
                   :aws_region,
-                  :output_location
+                  :output_location,
+                  :aws_profile
 
     def initialize
       @aws_access_key = nil
       @aws_secret_key = nil
       @aws_region = nil
       @output_location = nil
+      @aws_profile = nil
```

Comment:
Can you update the README file and document this change?  It'll be helpful for other people who are looking for this option.